### PR TITLE
wire printflags through run cmd

### PR DESCRIPTION
**Pull request by juanvallejo**
**Release note**:
```release-note
NONE
```

Adds PrintFlags to `run` command.

cc @soltysh @deads2k

### CODE REVIEW

**Comment by deads2k** [comment]
/assign @soltysh 
@kubernetes/sig-cli-maintainers

File: hack/make-rules/test-cmd-util.sh
Comment (juanvallejo) [inline hack/make-rules/test-cmd-util.sh]: @soltysh this was updated, as this PR defaults to using the new generator

File: pkg/kubectl/cmd/run.go
Comment (soltysh) [inline pkg/kubectl/cmd/run.go]: Once this merges, we should clean this and move them to `Validate`, but in a separate PR.

**Review by soltysh — APPROVED** [review]
/approve no-issue

**Comment by soltysh** [comment]
Rebase and I'll apply the lgtm label

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**
